src/utils.rs

```diff
@@ -166,8 +213,27 @@ impl Console {
                     )
                 })?;
             }
+
+            // Account for the read/write/epoll_wait elapsed time
+            epoll_timeout_us -= start_epoll_time.elapsed().as_micros() as i128;
         }
 
+        // Clear console read fd from epoll
+        epoll::epoll_ctl(epoll_fd, EpollOp::EpollCtlDel, self.fd, None).map_err(|e| {
+            new_nitro_cli_failure!(
+                &format!("Failed to remove fd from epoll: {:?}", e),
+                NitroCliErrorEnum::EpollError
+            )
+        })?;
+
+        // Close epoll fd
+        unistd::close(epoll_fd).map_err(|e| {
```

Comment:
I don't expect this is to be needed, why do you think is needed.
Normally the ownership of the underlying filedescriptor it is owned by the EpollFd struct and when that goes out of scope, it will close the fd as well.